.storybook/config.js

```diff
@@ -1,18 +1,23 @@
-import {configure, addDecorator, addParameters, forceReRender} from '@storybook/react';
-import {DocsPage, DocsContainer} from '@storybook/addon-docs/blocks';
-import {withKnobs} from '@storybook/addon-knobs/react';
-import {create} from '@storybook/theming';
-import addons from '@storybook/addons';
-import Events from '@storybook/core-events';
-import {toId} from '@storybook/router';
-import ReactDOM from 'react-dom';
+import {
+  configure,
+  addDecorator,
+  addParameters,
+  forceReRender
+} from "@storybook/react";
+import { DocsPage, DocsContainer } from "@storybook/addon-docs/blocks";
+import { withKnobs } from "@storybook/addon-knobs/react";
+import { create } from "@storybook/theming";
+import addons from "@storybook/addons";
+import Events from "@storybook/core-events";
+import { toId } from "@storybook/router";
+import ReactDOM from "react-dom";
 
-import {commonColors, typeColors, fontFamily} from '../modules/core/react';
-import {InputProviderDecorator, FontsDecorator} from '../utils/storybook';
+import { commonColors, typeColors, fontFamily } from "../modules/core/react";
```

Comment:
Just noticed there's a bunch of weird formatting stuff going on in this file. Will fix.